libs/input-mapping/src/File/Strategy/AbstractStrategy.php

```diff
@@ -58,7 +58,17 @@ public function downloadFiles(array $fileConfigurations, string $destination): I
         $fileOptions->setFederationToken(true);
         $outputStateList = [];
         foreach ($fileConfigurations as $fileConfiguration) {
-            $files = Reader::getFiles($fileConfiguration, $this->clientWrapper, $this->logger, $this->fileStateList);
+            $fileOptionsRewritten = Reader::getFiles($fileConfiguration, $this->clientWrapper, $this->logger);
+            $options = $fileOptionsRewritten->getStorageApiFileListOptions($this->fileStateList);
+            if ($fileOptionsRewritten->getSourceBranchId() ===
```

Comment:
Tady se mě to bohužel nepodařilo vyřešit jako u tabulek, takže ifujeme - pro konkrétní konfigurací souborů se použije konkrétní klient